recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
Sometimes things will pick up `gcc`. This is a frontend to `clang` (use to be a proper compiler on Mac), but forces the ancient (and basically unsupported) 4.2.1 version of GCC and its libraries to be used. Explicitly setting `CC` and `CXX` will avoid that.